backend/routes/productRoutes.js

```diff
@@ -7,4 +7,20 @@ router.get('/', async (req, res) => {
     res.send(products);
 });
 
+router.post('/', (req, res) => {
+    // I pass the body of the request as argument of the Product constructor and assign to a variable
+    let product = new Product(req.body);
```

Comment:
Can you do some checks on the data to make sure it's valid?

Like is the price have a value? 
Does description have a non empty string. 

If it fails these validation we should not insert and instead return a 400 response

Reply:
Sure no problem